docs/simple-example.md

```diff
@@ -159,11 +87,46 @@ How `Secrets` and `ConfigMaps` are stored on disk mirror the storage for
 where a file is created for each key in a `ConfigMap` `data` map or `binaryData` map and each key in a `Secret`
 `data` map.
 
-If you want to try other `ConfigMaps` or a `Secret`, first clear out the existing application:
+## BuildConfig example
+
+You can simply start a `Build` for this `BuildConfig`, and follow the logs, via 
 
 ```shell
-$ oc delete -f ./examples 
-``` 
+$ oc start-build my-csi-bc -F
+```
+
+and what as Red Hat entitled content via your subscription is installed during the running of the Docker Strategy build.
+See the next section first however, if you have never used OpenShift Insights Operator to inject your subscription 
+credentials into your cluster.
+
+## Notes or setting up your cluster so that Insights Operator injects your subscription credentials for use in your cluster
+
+So both OpenShift Shared Resources and OpenShift Insights Operator are only available on Tech Preview clusters.
```

Comment:
Starting a sentence with "So" is very informal, and isn't necessary. In addition, the Insights Operator itself is available on all clusters - only the "download Simple Content Accesss cert" is tech preview.

```suggestion
Tech Preview must be enabled on your OCP cluster to use the Shared Resources CSI Driver and to sync the Simple Content Access certificate of your Red Hat account.
```